tests/lib/rules/no-invalid-html-attribute.js

```diff
@@ -537,6 +560,74 @@ ruleTester.run('no-invalid-html-attribute', rule, {
         },
       ],
     },
+    {
+      code: '<link rel="shortcut"></link>',
+      output: '<link rel="shortcut icon"></link>',
```

Comment:
as commented; this isn't a safe autofix.
```suggestion
      code: '<link rel="shortcut"></link>',
```

Reply:
4de38ef